pythonarcade/arcade
# Improve scene object annotations & docstrings

### Changes

* Rewrite multiple docstrings for clarity & precision
* Add links to other relevant sections of arcade & python doc
* Add missing keyword arguments & type annotations to `Scene.draw`

### How to test

1. Build & serve doc locally with `./make.py serve`
2. Open http://localhost:8000/api_docs/api/sprite_scenes.html#arcade.Scene in your browser
3. Check for anything wrong

### Progress
- [x] Top-level docstring
- [x] Sprite list addition docstrings
- [x] Dunder docstrings
- [x] Drawing docstrings
- [x] `get_*` docstrings
- [x] Take care of `move_*` function docstrings
- [x] Find & fix any remaining docstrings & annotation in need of attention
- [x] Double check backtick formatting for variables displaying as italics
